sdk/data-generators/private-set-intersection-sum-generator/src/main.rs

```diff
@@ -100,10 +100,23 @@ fn main() -> Result<(), Box<dyn Error>> {
     let mut rng = StdRng::seed_from_u64(seed);
 
     for i in 0..iter {
-        let input = generate(&mut rng, size as usize);
-        let bytes = pinecone::to_vec(&input).unwrap();
-        let path = dir.join(format!("data-{}-{}.dat", size, i));
-        File::create(path)?.write(bytes.as_slice())?;
+        let (data, sample) = generate(&mut rng, size as usize);
+        let data = pinecone::to_vec(&data).unwrap();
```

Comment:
Can we have some more elegant error handling here, rather than `unwrap()` failing?  e.g. use `map_err()` and `eprintln!()` to explain what went wrong and kill the program with an appropriate error code.